Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml

```diff
@@ -29702,4 +29719,18 @@
         <GlossaryPriority value="201"/>
         <HotkeyCategory value="Unit/Category/AP_ZergUnits"/>
     </CUnit>
+    <CUnit id="AP_BanelingLaunchAmmo">
+        <FlagArray index="Uncommandable" value="1"/>
+        <FlagArray index="Unselectable" value="1"/>
+        <FlagArray index="Untargetable" value="1"/>
+        <FlagArray index="UseLineOfSight" value="1"/>
+        <FlagArray index="Invulnerable" value="1"/>
+        <FlagArray index="AILifetime" value="1"/>
+        <FlagArray index="ArmySelect" value="1"/>
+        <EditorFlags index="NoPlacement" value="1"/>
+    </CUnit>
+    <CUnit id="AP_BanelingLaunchProjectile" parent="MISSILE">
+        <Race value="Zerg"/>
+        <EditorCategories value="ObjectFamily:Campaign,ObjectType:Projectile"/>
+    </CUnit>

```

Comment:
Should Raven's PDD be able to shot down this? Spells often use `MISSILE_INVULNERABLE` as a parent

Reply:
Swapped parent to MISSILE_INVULNERABLE